tests/feature-store/test_model.py

```diff
@@ -49,30 +49,40 @@ def test_features_parser():
 
 
 @pytest.mark.parametrize(
-    "raise_error, pass_run_id, is_single_file, target_path",
+    "pass_run_id, is_single_file, target_path, expected_path",
     [
-        (False, True, False, "v3io:///bigdata/{run_id}"),
-        (False, True, True, "v3io:///bigdata/{run_id}/file.parquet"),
-        (False, False, False, "v3io:///bigdata/"),
-        (False, False, True, "v3io:///bigdata/file.parquet"),
-        (False, True, False, "v3io:///bigdata/"),
-        (False, True, True, "v3io:///bigdata/file.parquet"),
-        (True, False, False, "v3io:///bigdata/{run_id}"),
-        (True, False, True, "v3io:///bigdata/{run_id}/file.parquet"),
+        (True, False, "v3io:///bigdata/{run_id}", "v3io:///bigdata/run_id_val"),
```

Comment:
Need to verify whether in this case (run_id in the path) we need to add `/` at the end of the path.